src/util/recorder.ts

```diff
@@ -577,23 +596,30 @@ export class Recorder {
 
     let streamingConsume = false;
 
+    const resourceType = reqresp.resourceType;
+
     const contentType = this._getContentType(responseHeaders);
 
     // set max fetch size higher for HTML responses for current page
-    const matchFetchSize = this.allowLargeContent(contentType)
+    const maxFetchSize = this.allowLargeContent(resourceType, contentType)
       ? MAX_BROWSER_TEXT_FETCH_SIZE
       : MAX_BROWSER_DEFAULT_FETCH_SIZE;
 
-    if (contentLen < 0 || contentLen > matchFetchSize) {
-      const opts = {
+    if (contentLen < 0 || contentLen > maxFetchSize) {
+      logger.warn(
```

Comment:
I'm seeing a lot of these in the logs, maybe should be `debug`? It gets quite verbose as-is.

Reply:
Yes, not really warning